BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -176,28 +176,8 @@ class BraintreeClient @VisibleForTesting internal constructor(
      * @suppress
      */
     fun sendGET(url: String, responseCallback: HttpResponseCallback) {
-        if (authorization is InvalidAuthorization) {
-            responseCallback.onResult(null, createAuthError())
-            return
-        }
-        getConfiguration { configuration, configError ->
-            if (configuration != null) {
-                httpClient.get(url, configuration, authorization) { response, httpError ->
-                    response?.let {
-                        try {
-                            sendAnalyticsTimingEvent(url, response.timing)
-                            responseCallback.onResult(it.body, null)
-                        } catch (jsonException: JSONException) {
-                            responseCallback.onResult(null, jsonException)
-                        }
-                    } ?: httpError?.let { error ->
-                        responseCallback.onResult(null, error)
-                    }
-                }
-            } else {
-                responseCallback.onResult(null, configError)
-            }
-        }
+        val request = BraintreeHttpRequest(method = "GET", path = url)
```

Comment:
Can we extract the method names into an enum?

Reply:
Yeah def. I can add that part in.